### ISSUE TITLE
Error while trying to login to the Dashboard profile of WSO2 API Manager Analytics 3.0.0

### ISSUE THREAD

**Issue by chaamini** [wso2/analytics-apim#953]
While running the dashboard profile from the default pack of WSO2 APIM analytics 3.0.0 I am getting the following error when I tried to login to the analytics dashboard. Please note I was able to successfully run the analytics worker profile and the API Manager 3.0.0 before starting the dashboard profile. I have not changed any configurations other than enabling analytics in the deployment.toml of API Manager.

[2020-02-05 13:43:03,310] ERROR {org.wso2.analytics.apim.idp.client.ApimIdPClient} - Error occurred while creating Login admin Service Client.

Please guide me on how to get rid of this issue
![image](https://user-images.githubusercontent.com/4281345/73824577-693d4d80-4820-11ea-97fc-bbeac2bfcd79.png)

**Comment by RAVEENSR** [comment]
Have change change the default certificates?  Also before running dashboard profile make sure that the API Manager 3.0.0 is full up and running.

**Comment by chaamini** [comment]
I made sure that the API Manager 3.0.0 was full up and running before running the dashboard profile. I did not change the default certificates as well.

**Comment by RAVEENSR** [comment]
Were you able to log into publisher in Apim 3.0?

**Comment by chaamini** [comment]
I was able to log in to publisher/devportal without any issue

**Comment by ruks** [comment]
@chaamini is this issue still persisting? Based on the error log it seems APIM endpoint is not available.

**Comment by chaamini** [comment]
Yes I even tried with the latest WUM updated pack of analytics 3.0.0 where I am getting the same issue.

**Comment by PrabakaranTS** [comment]
Is there any solution for this? 

I am also facing the same issue with 3.0.0

**Comment by ruks** [comment]
The possible cause for the issues are either APIM not fully updating running or APIM hostname or port is different.
